src/content/docs/en/guides/integrations-guide/partytown.mdx

```diff
@@ -149,6 +150,57 @@ export default defineConfig({
 });
 ```
 
+### config.resolveUrl
+
+Some third-party scripts may require [proxying](https://partytown.qwik.dev/proxying-requests/) through `config.resolveUrl()`, which runs inside the service worker. You can set this configuration option to check for a specific URL, and optionally return a proxied URL instead:
+
+```js title="astro.config.mjs" {7-13}
+export default defineConfig({
+  // ...
+  integrations: [
+    partytown({
+      // Example: proxy Facebook's analytics script
+      config: {
+        resolveUrl: (url) => {
+          const proxyMap = {
+            "connect.facebook.net": "my-proxy.com"
+          }
+          url.hostname = proxyMap[url.hostname] || url.hostname;
+          return url;
+        },
+      }
+    }),
+  ],
+});
+```
+
+However since the `config` object is serialized when sent to the client, some limitations on functions passed to your configuration apply:
+
+- Functions cannot reference anything outside of the function scope.
+- Functions can only be written in JavaScript.
+
+In some advanced use cases, you may need to pass data to this function while initializing Partytown. To do so, you can set `resolveUrl()` on `window.partytown` instead of the integration config:
+
+```astro title="Head.astro" {8-14}
+---
+const proxyMap = {
+  "connect.facebook.net": "my-proxy.com"
+};
+---
+
+<script is:inline set:html={`
+window.partytown = {
+  resolveUrl: (url) => {
+    const proxyMap = ${JSON.stringify(proxyMap)};
+    url.hostname = proxyMap[url.hostname] || url.hostname;
+    return url;
+  },
+};
```

Comment:
```suggestion
	window.partytown = {
  	resolveUrl: (url) => {
    	const proxyMap = ${JSON.stringify(proxyMap)};
    	url.hostname = proxyMap[url.hostname] || url.hostname;
    	return url;
  	},
	};
```
I just indented everything by one tab, because the preview was showing this: 
<img width="481" height="352" alt="image" src="https://github.com/user-attachments/assets/c29c5c5e-b38a-4d7b-b084-5b04014657cf" />